CONTRIBUTING.md

```diff
@@ -497,6 +497,88 @@ rebase the branch on main, fixing any conflicts along the way before the code ca
 
 ---
 
+## Adding a new ACL Token
+
+Checklist for getting server-acl-init to generate a new ACL token assuming the new token
+is named `foo`.
+
+### Control Plane
+
+* `control-plane/subcommand/server-acl-init/command.go`
+    * Add `flagCreateFooToken bool` to vars list
+    * Initialize flag in `init`
+
+      ```go
+      c.flags.BoolVar(&c.flagCreateFooToken, "create-foo-token", false,
+        "<docs for flag>")
+      ```
+    * Add `if` statement in `Run` to create your token (follow placement of other tokens).
+      You'll need to decide if you need a local token (`createLocalACL`) or global token `createGlobalACL`.
+      
+      ```go
+      if c.flagCreateFooToken {
+          err := c.createLocalACL("foo", fooRules, consulDC, isPrimary, consulClient)
+          if err != nil {
+              c.log.Error(err.Error())
+              return 1
+          }
+      }
+      ```
+* `control-plane/subcommand/server-acl-init/rules.go`
+    * Add a function that outputs your rules using a template
+      (if the rules are always the same us a `const`):
```

Comment:
```suggestion
      (if the rules do not change, use a `const`):
```

"same" should be used when it's explicit what it is the same as. This is implicitly the same as it was in the past so it's easier to just say it doesn't change.